SampleXCUITests/SampleXCUITests.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  SampleXCUITests.swift
+//  SampleXCUITests
+//
+//  Created by Pankaj Yadav on 12/04/23.
+//  Copyright © 2023 BrowserStack. All rights reserved.
+//
+
+import XCTest
+import PercyXcui
+
+class SampleXCUITests: XCTestCase {
+        
+    override func setUp() {
+        super.setUp()
+        
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+        
+        // In UI tests it is usually best to stop immediately when a failure occurs.
+        continueAfterFailure = false
+        // UI tests must launch the application that they test. Doing this in setup will make sure it happens for each test method.
+        XCUIApplication().launch()
+
+        // In UI tests it’s important to set the initial state - such as interface orientation - required for your tests before they run. The setUp method is a good place to do this.
+    }
+    
+    override func tearDown() {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+        super.tearDown()
+    }
+    
+    func testText() {
+        let app = XCUIApplication()
+        let appPercy = AppPercy()
+        
+        do {
+            try appPercy.screenshot(name: "First Screenshot")
+        } catch {
+            NSLog("App percy screenshot failed")
+        }
+        
+        //visit the text page
+        app.buttons["Text"].tap()
+        
+        let enterText = "Hi Browserstack!!"
+        
+        do {
+            try appPercy.screenshot(name: "Second Screenshot")
```

Comment:
can we have screenshot options passed in second screenshot. just to show the usage.